src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -5,136 +5,139 @@
       "Size": 3568
     },
     "assemblies/_Microsoft.Android.Resource.Designer.dll": {
-      "Size": 1942
+      "Size": 1940
     },
     "assemblies/FormsViewGroup.dll": {
       "Size": 7313
     },
     "assemblies/Java.Interop.dll": {
-      "Size": 66786
+      "Size": 66792
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 444703
+      "Size": 445418
     },
     "assemblies/Mono.Android.Runtime.dll": {
-      "Size": 5895
+      "Size": 5825
     },
     "assemblies/mscorlib.dll": {
-      "Size": 3857
+      "Size": 3850
     },
     "assemblies/netstandard.dll": {
-      "Size": 5567
+      "Size": 5562
     },
     "assemblies/rc.bin": {
       "Size": 1182
     },
     "assemblies/System.Collections.Concurrent.dll": {
-      "Size": 10529
+      "Size": 10520
     },
     "assemblies/System.Collections.dll": {
-      "Size": 15258
+      "Size": 15249
     },
     "assemblies/System.Collections.NonGeneric.dll": {
-      "Size": 7487
+      "Size": 7479
     },
     "assemblies/System.ComponentModel.dll": {
-      "Size": 1964
+      "Size": 1958
     },
     "assemblies/System.ComponentModel.Primitives.dll": {
-      "Size": 2579
+      "Size": 2573
     },
     "assemblies/System.ComponentModel.TypeConverter.dll": {
-      "Size": 6069
+      "Size": 6066
     },
     "assemblies/System.Console.dll": {
-      "Size": 6610
+      "Size": 6601
     },
     "assemblies/System.Core.dll": {
-      "Size": 1982
+      "Size": 1975
     },
     "assemblies/System.Diagnostics.TraceSource.dll": {
-      "Size": 6580
+      "Size": 6576
     },
     "assemblies/System.dll": {
-      "Size": 2338
+      "Size": 2330
     },
     "assemblies/System.Drawing.dll": {
-      "Size": 2023
+      "Size": 2016
     },
     "assemblies/System.Drawing.Primitives.dll": {
-      "Size": 11996
+      "Size": 11990
+    },
+    "assemblies/System.IO.Compression.Brotli.dll": {
+      "Size": 11852
     },
```

Comment:
Is there a way to avoid the new assembly? It seems like any app that uses `HttpClient` will now bring in `BrotliStream`? Maybe that is OK?

If we added some feature flag, then you'd have to enable a setting to use it. I'm not sure if that is better.

Reply:
The assembly is so tiny that I don't think it's problem, especially that the native library is 700k and included anyway (it contains all the System.IO.Compression native code to handle Deflate, GZip and Brotli).